tests/generate_code.bat

```diff
@@ -29,7 +29,7 @@ IF NOT "%MONSTER_EXTRA%"=="skip" (
 )
 
 cd ../samples
-..\%buildtype%\flatc.exe --cpp --lobster --gen-mutable --reflect-names --gen-object-api --gen-compare --cpp-ptr-type flatbuffers::unique_ptr monster.fbs || goto FAIL
+..\%buildtype%\flatc.exe --cpp --lobster --gen-mutable --reflect-names --gen-object-api --gen-compare --cpp-ptr-type flatbuffers::unique_ptr --cpp-include "string" --cpp-include "vector" monster.fbs || goto FAIL
```

Comment:
We can probably leave this out, as I'd prefer the sample code not to have headers that are not actually needed.

Reply:
fixed